test/package/test_freeze.py

```diff
@@ -0,0 +1,47 @@
+import textwrap
```

Comment:
We'll need to make a few changes to the structure to support a pure Python test:
1. since this test covers `torch::deploy` and not `torch.package`, let's create a new `test/test_deploy.py` file and move that code there. You don't have to replicate the complicated structure fo the `test/package/` tree, since I don't think we'll have that many pure Python deploy tests—just a single general file called `test_deploy.py` is sufficient.
2. I'm not sure, but I don't think `torch/csrc/…` is technically guaranteed to be in PyTorch's Python install tree, since it's a not public library path (it's for C src). Let's move the `freeze.py` script to somewhere more conventional, like `torch/utils/_freeze.py` (note the forward underscore as freezing is not a user-facing module).

Reply:
Done. I've also changed the CMakeLists.txt under torch/csrc/deploy/interpreter. That cmake file will refer to the freeze script to create frozen python modules. Since we move the freeze script, we need change the commands in the cmake file accordingly.